app/models/comment.rb

```diff
@@ -1,5 +1,5 @@
 class Comment < ApplicationRecord
-  belongs_to :author, class_name: 'User'
+  belongs_to :user, class_name: 'User'
   belongs_to :post
 
```

Comment:
- [ ] The way you defined the relationship between `Comment` and `Post` is alright. But the migration file references `comment_id` instead of `post`. All you need to do is `t.references :post, foreign_key: true, null: false` in the column that references post. 👍